tests/step-018.sh

```diff
@@ -14,16 +14,18 @@ mkdir -p /tmp/shared/hooks-018.git/pre-commit &&
 mkdir -p /tmp/test18 && cd /tmp/test18 || exit 1
 git init || exit 1
 
+git config githooks.sharedHooksUpdateTriggers ".githooks.shared.trigger"
+
 mkdir -p .githooks &&
     echo '/tmp/shared/hooks-018.git' >.githooks/.shared &&
     HOOK_NAME='.githooks.shared.trigger' \
         sh ~/.githooks/release/base-template-wrapper.sh ||
     exit 1
 
-HOOK_NAME=pre-commit HOOK_FOLDER=$(pwd)/.git/hooks \
-    sh ~/.githooks/release/base-template-wrapper.sh
+OUT=$(HOOK_NAME=pre-commit HOOK_FOLDER=$(pwd)/.git/hooks \
+    sh ~/.githooks/release/base-template-wrapper.sh)
 
-if [ $? -ne 1 ]; then
+if ! echo "$OUT" | grep "Fail here"; then
```

Comment:
This just makes sure we have started the shared hook execution, but it doesn't check if it's failed as expected (by the original test). Is this change correct?